app/_how-tos/get-started-with-metering-and-billing.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Get started with Metering and Billing in {{site.konnect_short_name}}
+description: Learn how to...
+content_type: how_to
+
+permalink: /metering-and-billing/get-started/
+breadcrumbs:
+  - /metering-and-billing/
+
+products:
+    - gateway
+    - metering-and-billing
+
+works_on:
+    - konnect
+
+tags:
+    - get-started
+
+tldr: 
+  q: What is Metering and Billing in {{site.konnect_short_name}}, and how can I get started with it?
+  a: |
+    blah
+
+tools:
+    - deck
+  
+prereqs:
+  skip_product: true
+  inline:
+    - title: cURL
+      content: |
+        [cURL](https://curl.se/) is used to send requests to {{site.base_gateway}}. 
+        `curl` is pre-installed on most systems.
+  entities:
+    services:
+        - example-service
+    routes:
+        - example-route
+
+cleanup:
+  inline:
+    - title: Clean up Konnect environment
+      include_content: cleanup/platform/konnect
+      icon_url: /assets/icons/gateway.svg
+
+min_version:
+    gateway: '3.4'
+next_steps:
+  - text: See all {{site.base_gateway}} tutorials
+    url: /how-to/?products=gateway
+  - text: Learn about {{site.base_gateway}} entities
+    url: /gateway/entities/
+  - text: Learn about how {{site.base_gateway}} is configured
+    url: /gateway/configuration/
+  - text: See all {{site.base_gateway}} plugins
+    url: /plugins/
+automated_tests: false
+---
+
```

Comment:
Should we add an intro like:
"This getting-started guide shows how you can meter Kong API requests and invoice your customers after their API consumption."